### ISSUE TITLE
[BUG]: Urban Reign: Graphical Glitch from blur effect (ex when winning a mission)

### ISSUE THREAD

**Issue by TytusDeZoo3123** [PCSX2/pcsx2#10763]
### Describe the Bug

This is pretty similiar to a bug I know troubled this game before on hardware mode, when a bunch of rainbow squares would pop up on the screen when for example you'd beat a mission. Apparently this was "fixed" however the issue is still present on hardware mode in a different way - its a bit odd and I did not have time to make a screenshot of it, but it basically looked like some black squares appearing on part of the screen and I think there was even text(?) there, odd

### Reproduction Steps

Pretty easy, you can see it happen even on the first mission - it will happen during the intro cutscene and when you win the mission, I'm pretty sure it happens when the game is trying to activate a blur effect

### Expected Behavior

The game should have a blur effect for a moment, not graphical glitches on the screen

### PCSX2 Revision

1.7.5537

### Operating System

Windows 10 (64bit)

### CPU

Intel Core i5 9300H @ 2.40GHz

### GPU

3071MB NVIDIA GeForce GTX 1050 (Lenovo)

### GS Settings

Nothing changed in the default graphics settings besides raising the resolution to 1080p, vulkan renderer

**Comment by refractionpcsx2** [comment]
Can you provide a GS dump please :)

**Comment by TytusDeZoo3123** [comment]
Here you go
The first one is the glitch that happens during the in-game intro to mission 1 when the protagonist starts fighting the 2 gangsters
For the second one I was a bit late since I had to quickly drop my controller, but you can still see the glitch fading out

![Urban Reign_SLUS-21209_20240202111558](https://github.com/PCSX2/pcsx2/assets/154998608/684ca34a-b7ff-4e26-911b-21b30e09f0d6)
![Urban Reign_SLUS-21209_20240202111746](https://github.com/PCSX2/pcsx2/assets/154998608/bc5aae4f-aa39-492f-80e7-0dd407c25890)

**Comment by refractionpcsx2** [comment]
I need a GS dump, not a screenshot :P the default keys are Shift + F8, but you can check it in the hotkeys setting, if that doesn't work.

Thanks

**Comment by TytusDeZoo3123** [comment]
sorry, sorry I'm dumb
https://files.catbox.moe/6hxbfc.zst
https://files.catbox.moe/dttwb2.zst

**Comment by Mrlinkwii** [comment]
gs dumps seems fine here 

![image](https://github.com/PCSX2/pcsx2/assets/17685135/eabd5da5-125a-4951-a7c7-1b3e50eadc3c)

![image](https://github.com/PCSX2/pcsx2/assets/17685135/94354012-2188-47f7-be5f-1eec641da848)

**Comment by Mrlinkwii** [comment]
@TytusDeZoo3123 if you load the gs dump , do you the same issue ?

**Comment by TytusDeZoo3123** [comment]
uhhhhhhhhhhhhh okay it's been a while but let me check, I'll have to reinstall pcsx2

**Comment by TytusDeZoo3123** [comment]
okay I tried again and the issue still does happen on vulkan HOWEVER I switched renderers to dx11 and the issue went away
so thats good I suppose

**Comment by refractionpcsx2** [comment]
did you try the GS dump you uploaded as we asked, instead of the game?

We need to know if you can reproduce it with the dump you gave us

**Comment by TytusDeZoo3123** [comment]
yes, I don't know how certain of a fix that is but the game works now if I use a different renderer
But now I don't know what is going on but there's something strange going on with the mission text that makes it hard to read, on native resolution its really pixelated and on higher resolution there's still some weird blur/ghosting(?) to it
I checked gameplay of the game and the text looked fine there
![Urban Reign_SLUS-21209_20240410234951](https://github.com/PCSX2/pcsx2/assets/154998608/2fde26cf-680a-482c-9a41-407b696065ad)
https://files.catbox.moe/y2f8gj.zst
(probably shouldn't use catbox because it removes the .gs from the files)

**Comment by lightningterror** [comment]
Reproduced it here, you'll have to keep switching between hw and sw on the dump to trigger it, garbage can vary each time

![Image](https://github.com/user-attachments/assets/e8d1315b-c2e3-49bc-a0d9-47fb76a2ddea)
![Image](https://github.com/user-attachments/assets/d5925cdd-bfe8-4a79-9750-1272ce61d24e)

**Comment by refractionpcsx2** [comment]
Can this be tested on #11461 (or after it merges)? Some stuff got fixed by that PR with this game I believe, so the glitches may be resolved.

**Comment by Virtua64** [comment]
Seems to be fixed with the RT in RT PR.

SW:
![Image](https://github.com/user-attachments/assets/d00e3d42-d641-433a-aa07-7addcb540e27)


PR HW: 
![Image](https://github.com/user-attachments/assets/3b6ebc98-9a6d-4a13-93cf-31366a8ab425)

Master HW:
![Image](https://github.com/user-attachments/assets/9d1a2c5a-3e9a-444e-9652-bc0109036c5f)


Also notice how the lighting in the master HW is muted whereas the PR HW lighting now looks correct like SW mode.


GS dump: 
[dttwb2.zip](https://github.com/user-attachments/files/19269791/dttwb2.zip)

**Comment by refractionpcsx2** [comment]
Thanks, I missed linking this issue in that PR :)
